Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -509,6 +509,89 @@ $ adb install UnCrackable-Level1.objection.apk
 $ objection explore
 ```
 
+#### Basic Information Gathering
+
+As mentioned previously, Android runs on top of a modified Linux kernel. Android retains the [*procfs*](https://www.kernel.org/doc/Documentation/filesystems/proc.txt "procfs") from Linux, and this filesystem is mounted at */proc*.  The *procfs* filesystem provides a directory-based view of process running on the system, providing information about the processes, threads, and other system-wide diagnostics. *procfs* is arguably one of the most important filesystem on Android. Many Android tools depend on *procfs* as the source of information.
```

Comment:
```suggestion
As mentioned previously, Android runs on top of a modified Linux kernel. Android retains the [proc filesystem](https://www.kernel.org/doc/Documentation/filesystems/proc.txt "procfs") from Linux, and this filesystem is mounted at /proc.  The proc filesystem (procfs) provides a directory-based view of process running on the system, providing information about the processes, threads, and other system-wide diagnostics. Procfs is arguably one of the most important filesystems on Android. Many Android tools depend on procfs as the source of information.
```

Bold is only used for UI navigation, see also https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md#navigation

Reply:
My bad on this one, I wanted to make it italics.